spec/models/claim/cleaner_spec.rb

```diff
@@ -0,0 +1,75 @@
+require "rails_helper"
+
+RSpec.describe Claim::Cleaner, type: :model do
+
+  let(:over_two_months_ago) { 2.months.ago - 1.day }
+
+  it "does not delete details from a newly submitted claim" do
```

Comment:
I think this spec and the spec below should create claims that are old enough to qualify for scrubbing, otherwise we are missing coverage. We should also have specs verifying that rejected and paid claims do not get scrubbed if they are not old enough.